ts/Core/Tooltip.ts

```diff
@@ -1042,7 +1042,7 @@ class Tooltip {
             styledMode = chart.styledMode;
         let textConfig = {} as Tooltip.FormatterContextObject;
 
-        if (!options.enabled) {
+        if (!options.enabled || !splat(pointOrPoints)[0].series) { // #16820
```

Comment:
We already do `splat(pointOrPoints)` on line 1037, and refer to item 0 on line 1038, so we can simplify this line to `if (!options.enabled || !point.series) { // #16820`